reference/config_files/global_conf.rst

```diff
@@ -36,6 +36,91 @@ have priority over globally defined ones in *global.conf*, and can be defined as
     tools.build:jobs=10
 
 
+Configuration file template
+++++++++++++++++++++++++++++
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+It is possible to use **jinja2** template engine for *global.conf*. When Conan loads this file, immediately parses
+and renders the template, which must result in a standard tools-configuration text.
+
+  .. code:: jinja
+
+     # Using all the cores automatically
+     tools.build:jobs={{os.cpu_count()}}
+     # Using the current OS
+     user.myconf.system:name = {{platform.system()}}
+
+
+.. note::
+
+    The Python packages passed to render the template are only ``os`` and ``platform``.
+
+
+Configuration data types
+++++++++++++++++++++++++++
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+All the values will be interpreted by Conan as the result of the python built-in `eval()` function:
+
+.. code-block:: text
+
+    # String
+    tools.microsoft.msbuild:verbosity=Diagnostic
+    # Boolean
+    tools.system.package_manager:sudo=True
+    # Integer
+    tools.microsoft.msbuild:max_cpu_count=2
+    # List of values
+    user.myconf.build:ldflags=["--flag1", "--flag2"]
+    # Dictionary
+    tools.microsoft.msbuildtoolchain:compile_options={"ExceptionHandling": "Async"}
+
+
+Configuration data operators
++++++++++++++++++++++++++++++++++
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+It's also possible to use some extra operators when you're composing different tools configurations:
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    # Defining several lists
+    user.myconf.build:ldflags=["--flag1 value1"]
+    user.myconf.build:cflags=["--flag1 value1"]
+
+    # Appending values into the existing list
+    user.myconf.build:ldflags+=["--flag2 value2"]
+
+    # Unsetting the existing value (it'd be like we define it as an empty value)
+    user.myconf.build:cflags=!
+
+    # Prepending values into the existing list
+    user.myconf.build:ldflags=+["--prefix prefix-value"]
+
+
+Running, for instance, :command:`conan install . -pr myprofile`, the configuration output will be something like:
+
+.. code-block:: bash
+
+    ...
+    Configuration:
+    [settings]
+    [options]
+    [build_requires]
+    [env]
+    [conf]
+    user.myconf.build:cflags=!
+    user.myconf.build:ldflags=['--prefix prefix-value', '--flag1 value1', '--flag2 value2']
+    ...
+
+
```

Comment:
Maybe this could be under the title: "using conf in profiles" or something like that?